components/filemanager/toolbar.md

```diff
@@ -10,36 +10,403 @@ position: 15
 
 # FileManager Toolbar
 
-The Toolbar of the FileManager provides the following built-in commands:
+The FileManager includes a built-in Toolbar that contains several commands. You may use it as is or customize it to include your desired tools.
```

Comment:
Try to provide some more value in this introductory sentence, otherwise the reader may as well skip it.

```suggestion
The FileManager includes a built-in Toolbar that contains commands for adding new files and folders, sorting and changing the view, and more. You can use the toolbar as is or customize it to include the desired built-in or custom tools.
```